bql/semantic/expression.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/google/badwolf/bql/lexer"
 	"github.com/google/badwolf/bql/table"
+	litutils "github.com/google/badwolf/triple/literal"
```

Comment:
Just curios. What is the goal of the alias? I do not recall we are using elsewhere in the codebase.